src/middlewares/verifyToken.js

```diff
@@ -0,0 +1,33 @@
+import jwt from 'jsonwebtoken';
+
+const secret = process.env.SECRET_KEY;
+
+/**
+ *
+ *
+ * @param {object} req
+ * @param {object} res
+ * @param {*} next
+ * @returns {*} json or next
+ */
+const verifyToken = (req, res, next) => {
+  const bearerToken = req.headers.authorization;
+  if (!bearerToken) {
+    return res.status(401).json({
+      error: 'No token provided'
+    });
+  }
+  const tokenArray = bearerToken.split(' ');
+  const token = tokenArray[1];
```

Comment:
I feel lines 20 and 21 won't be needed here since we are getting the token from the authorization header. We can include it there directly when testing on Postman or mocha, which would mean there won't be any need for the `Bearer ` prefix.

Reply:
Changes made, thanks.